crawl/workspace.py

```diff
@@ -31,20 +31,10 @@ def __init__(self, repo_root_path, config, maven_install_info,
         self.dependency_metadata = dependency_metadata
         self.change_detection_enabled = config.change_detection_enabled
         self.override_file_info = override_file_info
-        self._name_to_ext_deps = self._parse_maven_install(maven_install_info, repo_root_path)
+        self.label_to_ext_deps = {}
+        self.external_dependencies = self._parse_maven_install(maven_install_info, repo_root_path)
```

Comment:
I think it is still good to make this a property, just because then you can better document what this is.  Like
```
@property
def external_dependencies(self):
    """
    Returns a tuple of all external dependencies (dependency.Dependency instances)  declared in pinned json files.
    """
```